src/apic-extension/HISTORY.rst

```diff
@@ -5,7 +5,15 @@ Release History
 
 1.0.0b5
 +++++
+* Add: Support yaml file for `az apic api register` command
+* Update: Update some command names, parameter names as well as related descriptions for better understanding. Please leverage `-h` option or refer Azure CLI reference doc to see full list of commands and parameters.
```

Comment:
@blackchoey :
* Update: Command names, parameter names, and command descriptions for better understanding. Please leverage `-h` option or refer Azure CLI reference doc to see full list of commands and parameters.